docs/generate_training_data.rst

```diff
@@ -0,0 +1,121 @@
+..
+    This file is part of INSPIRE.
+    Copyright (C) 2014-2018 CERN.
+
+    INSPIRE is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    INSPIRE is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+
+    In applying this license, CERN does not waive the privileges and immunities
+    granted to it by virtue of its status as an Intergovernmental Organization
+    or submit itself to any jurisdiction.
+
+==========================
+Generate the training data
+==========================
+
+We need annotated data to train the classifier in a supervised way. The data should contain title and abstract as well as their corresponding labels i.e. Rejected, Non-Core, or Core. We do have that data available in INSPIRE due to the large number of daily harvests and the corresponding curator actions to classify them in one of the aforementioned categories.
+
+The training data needs to be generated and organized in an appropriate format. The process, however, is somewhat complex. Data for Core and Non-Core records needs to be generated separately from data for the Rejected records.
+
+Generate Core and Non-Core data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Core and Non-Core records can be extracted from the INSPIRE dump data migrated on a local inspire-next instance. First, we would need to `setup inspire-next locally <https://inspirehep.readthedocs.io/en/latest/getting_started.html>`_. Next, we need to migrate the data from a recent `INSPIRE dump <http://inspirehep.net/dumps/inspire-dump.html>`_. The HEP dump is the relevant one in this case. Download that, and from the local inspire-next directory, go to the inspirehep terminal:
+
+::
+
+    docker-compose run --rm web bash
+
+In the terminal, you can migrate from the downloaded dump using:
+
+::
+
+    inspirehep migrate file -f <path_to_downloaded_file>
```

Comment:
* you want to add `-w` here, otherwise it won't wait for the migration to finish
* Instead of using `-f`, it's better to put `APP_DEBUG=False` in front to turn off debug mode.